requests.go

```diff
@@ -133,7 +135,12 @@ func (c *Client) copymove(method string, oldpath string, newpath string, overwri
 		log(fmt.Sprintf(" TODO handle %s - %s multistatus result %s", method, oldpath, String(data)))
 
 	case 409:
-		// TODO create dst path
+		err := c.createParentCollection(newpath)
+		if err != nil {
+			return err
+		}
+
+		return c.copymove(method, oldpath, newpath, overwrite)
```

Comment:
if something went wrong, you may stuck in a loop.

Reply:
I already thought about it, but I think _No, I can't_ because of [paragraph #6 section 9.8.5](https://tools.ietf.org/html/rfc4918#section-9.8.5) and [paragraph #6 section 9.9.4](https://tools.ietf.org/html/rfc4918#section-9.9.4) of [RFC 4918](https://tools.ietf.org/html/rfc4918)